File: 1-Beginner/1012/bee1012.java
// 1012 - Área
import java.io.IOException;
import java.util.Scanner;

class bee1012 {

    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);

        double a, b, c;
        a = scan.nextDouble();
        b = scan.nextDouble();
        c = scan.nextDouble();

        System.out.printf("TRIANGULO: %.3f\n", (a*c)/2);
        System.out.printf("CIRCULO: %.3f\n", 3.14159 * Math.pow(c, 2));
        System.out.printf("TRAPEZIO: %.3f\n", (a+b)*c/2);
        System.out.printf("QUADRADO: %.3f\n", Math.pow(b, 2));
        System.out.printf("RETANGULO: %.3f\n", a*b);

        scan.close();
    }
}

File: 1-Beginner/1012/bee1012.c
// 1012 - Área
#include <stdio.h>
#include <math.h>

int main(void) {

    double a, b, c;
    scanf("%lf%lf%lf", &a, &b, &c);

    printf("TRIANGULO: %.3lf\n",(a*c)/2);
    printf("CIRCULO: %.3lf\n", 3.14159 * pow(c, 2));
    printf("TRAPEZIO: %.3lf\n", (a+b)*c/2);
    printf("QUADRADO: %.3lf\n", pow(b, 2));
    printf("RETANGULO: %.3lf\n", a*b);

    return 0;
}